### PR TITLE
mute/unmute command

### PR DESCRIPTION
**Pull request by RedChrome**
Adds the command "mute" to mute and unmute the speakers. 

It's usefull when listening to online streams. Pausing/unpausing takes some time to start playing the stream again.

### CODE REVIEW

**Comment by nefthy** [comment]
Looks good to me. Please add documentation to the man page.

**Comment by RedChrome** [comment]
Right. Added the doc to the man page.

**Comment by nefthy** [comment]
@flyingmutant looks good

File: command_mode.c [hunk 1]
```diff
@@ -2585,6 +2612,7 @@ struct command commands[] = {
 	{ "version",               cmd_version,          0, 0,  NULL,                 0, 0          },
 	{ "view",                  cmd_view,             1, 1,  NULL,                 0, 0          },
 	{ "vol",                   cmd_vol,              1, 2,  NULL,                 0, 0          },
+    { "mute",                  cmd_mute,             0, 0,  NULL,                 0, 0          },
```
Comment (flyingmutant) [inline command_mode.c]: formatting is broken here (spaces instead of tabs)

File: command_mode.c [hunk 2]
```diff
@@ -1196,6 +1196,33 @@ static int parse_vol_arg(const char *arg, int *value, unsigned int *flags)
 	return -1;
 }
 
+static void cmd_mute(char *arg)
+{
+    static int l_prev = 0,  r_prev = 0;
```
Comment (flyingmutant) [inline command_mode.c]: please avoid funtion-level statics

File: command_mode.c [hunk 3]
```diff
@@ -1196,6 +1196,33 @@ static int parse_vol_arg(const char *arg, int *value, unsigned int *flags)
...
+    if (rc != OP_ERROR_SUCCESS) {
+        char *msg = op_get_error_msg(rc, "can't change volume");
+        error_msg("%s", msg);
+        free(msg);
+    } else {
+        mpris_volume_changed();
+    }
+    update_statusline();
+    return;
+}
```
Comment (flyingmutant) [inline command_mode.c]: please use tabs instead of spaces for indentation

Comment (flyingmutant) [reply command_mode.c]: also, `return;` at the end is a no-op, please drop it

**Comment by TUM-E21-ThinFilms** [comment]
Fixed the formatting and the other change requests.

File: command_mode.c [hunk 4]
```diff
@@ -2585,6 +2611,7 @@ struct command commands[] = {
 	{ "version",               cmd_version,          0, 0,  NULL,                 0, 0          },
 	{ "view",                  cmd_view,             1, 1,  NULL,                 0, 0          },
 	{ "vol",                   cmd_vol,              1, 2,  NULL,                 0, 0          },
+	{ "mute",                  cmd_mute,             0, 0,  NULL,                 0, 0          },
```
Comment (flyingmutant) [inline command_mode.c]: Sorry, did not spot this before: the list of commands is sorted, please keep it that way.

**Comment by TUM-E21-ThinFilms** [comment]
Sorted the list. Hope its fine now ;)

**Comment by flyingmutant** [comment]
Thanks! Merged.

**Pull request merged**